lib/components/Student/Student.js

```diff
@@ -23,10 +23,6 @@ class Student extends Component {
   async sendSummary() {
     const studentId = parseInt(this.props.match.params.id);
     const student = this.props.students[studentId].name;
-    // const summaryValues = Object.values(this.props.coreValues)
```

Comment:
getting rid of comments! 
![giphy 7](https://user-images.githubusercontent.com/6845268/32588167-195e2dba-c4ca-11e7-9f74-ece7540ef6ce.gif)